### DISCUSSION TITLE
AttributeError: 'API' object has no attribute 'search'

### DISCUSSION THREAD

**Discussion by OmkarGharat** [tweepy/tweepy#1709] [Questions] [answered]
I am getting error as AttributeError: 'API' object has no attribute 'search' , I don't know what is the reason behind this . Can anyone help me for this ??

`from textblob import TextBlob
import tweepy
import sys

from my_keys import mykeys

api_key = mykeys[0]
api_key_secret = mykeys[1]
access_token = mykeys[2]
access_token_secret = mykeys[3]

auth_handler = tweepy.OAuthHandler(consumer_key=api_key, consumer_secret=api_key_secret)

auth_handler.set_access_token(access_token, access_token_secret)
api = tweepy.API(auth_handler)

search_term = 'stocks'
tweet_amount = 200

tweets = tweepy.Cursor(api.search, q=search_term, lang='en').items(tweet_amount)

polarity = 0
positive = 0
negative = 0
neutral = 0

for tweet in tweets:
    final_text = tweet.text.replace('RT', '')
    if final_text.startswith('@'):
        position = final_text.index(':')
        final_text = final_text[position+2:]
    analysis = TextBlob(final_text)
    tweet_polarity = analysis.polarity
    if tweet_polarity > 0:
        positive += 1
    elif tweet_polarity < 0:
        negative += 1
    else:
        neutral += 1
    polarity += tweet_polarity
    print(final_text)  # tweets

print(polarity)
print(f'Amount of positive tweets : {positive}')
print(f'Amount of negative tweets : {negative}')
print(f'Amount of neutral tweets : {neutral}')
`

**Comment by Harmon758** [comment, marked as answer]
`API.search` was renamed to `API.search_tweets` in [Tweepy v4.0.0](https://github.com/tweepy/tweepy/releases/tag/v4.0.0).

For code block usage, see https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks.
